bookmark.html

```diff
@@ -4,8 +4,69 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <a href="./index.html">index</a>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kdam+Thmor+Pro&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
   </head>
-  <body></body>
+  <main>
+    <body>
+      <h1>BOOKMARKS</h1>
+      <section>
+        <article class="question-section">
+          <img
+            class="box-img"
+            src="images/bookmark_white_24dp(1).svg"
+            alt="bookmark-icon"
+          />
+          <h2>Question</h2>
+          <p>This is where questions being saved!</p>
+        </article>
+      </section>
+      <br />
+      <br />
+      <section>
+        <article>
+          <h2>Answer</h2>
+          <img
+            class="box-img2"
+            src="images/bookmark_white_24dp(1).svg"
+            alt="bookmark-icon"
+          />
+          <p>This is where questions being saved</p>
+        </article>
+      </section>
```

Comment:
In my opinion is the question and the answer part of one article or section. Therefore, I would recommend you to have each question card (see the wireframes) in one parent element. And decide if you want to work with `article` or `section` that is up to you, but you don't need both elements here.